### PR TITLE
RSDK-4317 RC card power sensor

### PR DESCRIPTION
**Pull request by oliviamiller**
wireframes: https://www.figma.com/file/XLsscBRMdg6kUKHVNSzXOA/Robot-Details-%3E-Config?type=design&node-id=4073-54630&mode=design&t=Uj3yFpt2D4HCgx3O-0

screenshot with fake sensor:
Screenshot 2023-08-09 at 11 30 19 AM

### CODE REVIEW

**Review by martha-johnston — APPROVED** [review]
lgtm! but I think you could request review from someone that does frontend to make sure

**Comment by micheal-parks** [comment]
Would it be possible to add this to `fake.json` so that it shows up when developing?

File: web/frontend/package-lock.json
Comment (micheal-parks) [inline web/frontend/package-lock.json:1]: can be removed since no package.json changes occurred

**Comment by oliviamiller** [comment]
> Would it be possible to add this to `fake.json` so that it shows up when developing?

yep its already in fake.json from another pr!

**Review by micheal-parks — APPROVED** [review]

**Pull request merged**